config/xml_schemas/config_machines_template.xml

```diff
@@ -72,14 +72,14 @@
     <CCSM_CPRNC>$ENV{CESMDATAROOT}/tools/cime/tools/cprnc/cprnc.cheyenne</CCSM_CPRNC>
 
     <!-- GMAKE: gnu compatible make tool, default is 'gmake' -->
-    <GMAKE></GMAKE>
+    <GMAKE>gmake</GMAKE>
 
     <!-- GMAKE_J: optional number of threads to pass to the gmake flag -->
     <GMAKE_J>8</GMAKE_J>
 
     <!-- BATCH_SYSTEM: batch system used on this machine,
       supported values are: none, cobalt, lsf, pbs, slurm -->
-    <BATCH_SYSTEM>pbs</BATCH_SYSTEM>
+    <BATCH_SYSTEM>none</BATCH_SYSTEM>
```

Comment:
With pbs, it seems like I would have needed a `config_batch.xml` entry for this dummy machine in order to use it in tests. To keep things simple, I changed the default to `none`.